### PR TITLE
List all PRs and get the master_commit from the PR by default

### PR DESCRIPTION
**Pull request by jrioux**
List all pull requests instead of only the first 30.

By default, determine the master commit based on the target of the pull request, thus obtaining accurate mergeable status.

### CODE REVIEW

**Comment by asmeurer** [comment]
I attempted to rewrite the code so that the default of `-m` was the commit that the pull request is against, and not master, but I found it to be a non-trivial change.  Glad to see you figured it out.

IMHO, "master_commit" should be renamed to "merge_commit".

File: sympy-bot [hunk 1]
```diff
@@ -420,6 +421,7 @@ def dispatch_reviews(config, urls, **kwargs):
         print "> Reviewing pull request #%d" % n
         repo_url = pull["head"]["repo"]["html_url"]
         repo_url = repo_url.replace(default_protocol, config.protocol)
+        master_commit = config.master_commit or "origin/" + pull["base"]["ref"]
```
Comment (asmeurer) [inline sympy-bot]: Does this do the right thing if the base repo is not sympy/sympy (I didn't check, so maybe it does)?

**Comment by jrioux** [comment]
I actually don`t know if it works with a repo different from sympy/sympy. How could we test it out?

**Comment by asmeurer** [comment]
`-R other/repo`

**Comment by jrioux** [comment]
It works quite nicely by specifying e.g. `-R mattpap/sympy` (no need to specify `-m`):
https://github.com/mattpap/sympy/pull/3

**Comment by asmeurer** [comment]
OK, I'm merging this then.

**Pull request merged**